crates/wasm-mutate/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 //! Wasm parser, validator, compiler, or any other Wasm-consuming
 //! tool. `wasm-mutate` can serve as a custom mutator for mutation-based
 //! fuzzing.
-
+#![feature(extend_one)]
```

Comment:
Let's avoid nightly-only features since none of the other projects in this repo rely on them.

Reply:
Solved!